gke-deploy/core/image/image.go

```diff
@@ -47,31 +40,32 @@ func ParseImages(ctx context.Context, images []string) ([]Image, error) {
 			return nil, fmt.Errorf("duplicate image name: %q", imName)
```

Comment:
off-topic: Are duplicates that big of a problem here? I don't know how this method is called, but without any more context I would have guessed this would just strip duplicates, rather than returning an error.

If this doesn't have to return an error, and `GetName` doesn't return any error, `ParseImages` can also not return an error.

Worth considering for a future PR.

Reply:
It is more of a nice-to-have. Allowing duplicates means the order of the image flag matters.
e.g., `-i image:1 -i image:2` will lead to containers pointing to image:2 and  `-i image:2 -i image:1` will lead to containers pointing to image:1

I can also opt into checking for duplicates at a higher layer closer to main.